documentation/Documentation.md

```diff
@@ -25,12 +24,12 @@ Documentation:
 * [Technical Specifications for the PMS integration](FHIRDocumentation.md)
 
 
-## Patient Focused Documentation
+## Patient-Focused Documentation

```

Comment:
```suggestion
## Patient-focused Documentation

```

Perhaps?

Reply:
Agreed. Note: http://www.future-perfect.co.uk/grammar-tip/is-it-focussed-or-focused/